internal/database/database.go

```diff
@@ -0,0 +1,14 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/uwblueprint/shoe-project/config"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func Connect() (*gorm.DB, error) {
+	dsn := fmt.Sprintf("user=%s password=%s dbname=%s port=%s sslmode=disable host=%s", config.GetDatabaseUser(), config.GetDatabasePassword(), config.GetDatabaseName(), config.GetDatabasePort(), config.GetDatabaseHost())
```

Comment:
this line seems too long. wrap it some where around 100 length but usually just wrap long lines in general since it improved code readability